INTITA/Modules/ForgotPassword/ForgotPasswordViewController.swift

```diff
@@ -7,11 +7,11 @@
 
 import UIKit
 
-enum NameCells: Int {
+enum ForgotCells: Int {
     case logoImageCell = 0
-    case passwordRecoveryCell
     case explanationLabelCell
     case emailTextFieldCell
+    case emptyCell
```

Comment:
What purposes is this empty cell for?

Reply:
I use multiple cells on different screens. An empty cell should make the screens look similar.